models/POP/README.rst

```diff
@@ -0,0 +1,440 @@
+##########
+POP README
+##########
+
+Contents
+========
+
+#. `Overview`_
+#. `Observations`_
+#. `Namelist`_
+#. `Files`_
+#. `Terms of Use`_
+
+Overview
+========
+
+The **Parallel Ocean Program (POP)** may be used with the **Data Assimilation
+Research Testbed (DART)**. Both main variants - `LANL POP
+<https://climatemodeling.science.energy.gov/projects/climate-ocean-and-sea-ice-modeling-cosim>`_
+and `CESM2.0 POP2 <https://www.cesm.ucar.edu/models/cesm2/ocean/>`_ have been
+tested.
+
+We are assimilating salinity and temperature observations from either the World
+Ocean Database (2005) or (2009).
```

Comment:
What about WOD13 ... and I thought we had some stuff from WOD17 ... although I have to figure out what/where.

Reply:
maybe dan (and/or ben) can help here?  the data providers update the entire WOD every 4 years, and then publish incremental updates quarterly or yearly (can't remember).  we've never dealt with anything but the whole archive.  they added some fields to the format in 13, i believe.  fred had a modified version of the converter code that read the updated format, but i had questions about whether it was actually using the updated field info correctly.  i don't know if any changes were made to the 17 formats that would require additional changes to the converter code or not.

Reply:
We do have WOD13 from Fred in `/glade/p/cisl/dares/Observations/WOD13`. I don't know of obs_seq files from WOD17 (although that doesn't preclude their existence).

The plan for this pull request is to merely convert the files from HTML to reStructuredText and then gather the team to revise the content within the files in a separate pull request.